datasets/monash_tsf/README.md

```diff
@@ -0,0 +1,234 @@
+| traffic_hourly                | Transport | 862           | 1H     | 48         | [Caltrans, 2020](http://pems.dot.ca.gov/)                                                                                           |
+| traffic_weekly                | Transport | 862           | 1W-WED | 8          | [Caltrans, 2020](http://pems.dot.ca.gov/)                                                                                           |
+| hospital                      | Health    | 767           | 1M     | 12         | [Hyndman, 2015](https://cran.r-project.org/web/packages/expsmooth/)                                                                 |
+| covid_deaths                  | Health    | 266           | 1D     | 30         | [Johns Hopkins University, 2020](https://github.com/CSSEGISandData/COVID-19)                                                        |
+| sunspot                       | Nature    | 1             | 1D     | 30         | [Sunspot, 2015](http://www.sidc.be/silso/newdataset)                                                                                |
+| saugeenday                    | Nature    | 1             | 1D     | 30         | [McLeod and Gweon, 2013](http://www.jenvstat.org/v04/i11)                                                                           |
+| us_births                     | Health    | 1             | 1D     | 30         | [Pruim et al., 2020](https://cran.r-project.org/web/packages/mosaicData)                                                            |
+| solar_4_seconds               | Energy    | 1             | 4S     | 60         | [Godahewa et al. 2021](https://openreview.net/pdf?id=wEc1mgAjU- )                                                                   |
+| wind_4_seconds                | Energy    | 1             | 4S     | 60         | [Godahewa et al. 2021](https://openreview.net/pdf?id=wEc1mgAjU- )                                                                   |
+| rideshare                     | Transport | 2304          | 1H     | 48         | [Godahewa et al. 2021](https://openreview.net/pdf?id=wEc1mgAjU- )                                                                   |
+| oikolab_weather               | Nature    | 8             | 1H     | 48         | [Oikolab](https://oikolab.com/)                                                                                                     |
+| temperature_rain              | Nature    | 32072         | 1D     | 30         | [Godahewa et al. 2021](https://openreview.net/pdf?id=wEc1mgAjU- )      
+
+
+### Dataset Usage
+
+To load a particular dataset just specify its name from the table above e.g.:
+
+```python
+load_dataset("monash_tsf", "nn5_daily")
+```
+> Notes:
+> - Data might contain missing values as in the original datasets.
+> - The prediction length is either specified in the dataset or a default value depending on the frequency is used as in the original repository benchmark.
+
```

Comment:
Maybe add an example if the users want to chanhe the `rolling_evaluations` or `prediction_length` ?